src/pages/Error404Page.vue

```diff
@@ -1,41 +1,208 @@
 <script>
-export default {
+import { defineComponent } from 'vue';
+
+export default defineComponent({
     name: 'Error404Page',
-};
+    computed: {
+        isMobile() {
+            return this.$q.screen.lt.md;
+        },
+        isLandscape() {
+            return this.$q.screen.width > this.$q.screen.height;
+        },
+    },
+    watch: {
+        $route: {
+            immediate: true,
+            handler() {
+                console.log(this.$q.screen);
+                console.log(this.$q.screen.lt);
+                console.log(this.$q.screen.lt.md);
+            },
+        },
+    },
+});
 </script>
 
 <template>
 <div
-    class="fullscreen bg-blue text-white text-center q-pa-md flex flex-center"
+    :class="{
+        'c-404-page': true,
+        'landscape' : isMobile && isLandscape
```

Comment:
to keep with BEM, this should be modified to `c-404-page--landscape`. that way no other styles from the rest of the site for `.landscape` affect this

Reply:
OMG! I keep on making newbie mistakes !! I'll pay more attention.